source/agora/common/Deserializer.d

```diff
@@ -284,6 +284,14 @@ public void deserializePart (T) (
         foreach (const ref field; record.tupleof)
             deserializePart(field, dg);
 
+    // Support for associative arrays / hashmap
```

Comment:
Ah, the serialization support was added in another PR, without the deserialization part?

Reply:
Ah, serialization, I knew I forgot something